cypress/page_objects/createBoard.js

```diff
@@ -0,0 +1,40 @@
+export default class CreateBoard {
+
+    get createNewBoardButton(){
+        return cy.get('li[title="Add new Board"]');
+    }
+
```

Comment:
delete empty row... be constant with that, either we place empty raw after every function or we dont